src/main/java/io/debezium/connector/spanner/db/DatabaseClientFactory.java

```diff
@@ -12,12 +12,14 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.google.api.gax.rpc.FixedHeaderProvider;
```

Comment:
Looks good, run a quick integration test with this change?

Reply:
Just ran a full integration test and all passed.